src/components/pages/Triggers/Triggers.tsx

```diff
@@ -114,31 +94,37 @@ const Triggers: React.FC = () => {
           return null;
         }
       });
+
       form.setFieldsValue({
         triggers: triggersData,
       });
+
       setDefaultFormattedTriggers(triggersData);
+    } else {
+      form.setFieldsValue({
+        triggers: [],
+      });
     }
   };
 
   useEffect(() => {
-    setDefaultTriggersData(triggersList);
+    setDefaultTriggersData(triggersList || []);
   }, [triggersList]);
 
   useEffect(() => {
     safeRefetch(refetch);
   }, []);
 
   const testsData = useMemo(() => {
-    return testsList.map((item: TestWithExecution) => ({
+    return testsList.map(item => ({
       name: item.test.name,
       namespace: item.test.namespace,
       type: item.test.type,
     }));
   }, [testsList]);
 
   const testSuitesData = useMemo(() => {
-    return testsSuitesList.map((item: any) => ({
+    return testsSuitesList.map(item => ({
```

Comment:
can we type this to an testsuite?

Reply:
It's already a `TestSuiteWithExecution` item

<img width="459" alt="image" src="https://user-images.githubusercontent.com/25991946/234315703-e7c4c6ae-5b40-4b1e-ac1e-712594a2857e.png">